specs/windows/interface_details.table

```diff
@@ -0,0 +1,33 @@
+table_name("interface_details")
```

Comment:
Man this really sucks : ( It would be nice to have the table name be the same but with different columns.